api-v2/models/models.py

```diff
@@ -1,37 +1,43 @@
 import os
 import psycopg2
 import config
-from werkzeug.security import generate_password_hash, check_password_hash
+from flask import jsonify
+from flask_jwt_extended import get_jwt_identity
+from werkzeug.security import generate_password_hash
 
-class Database:
-    def __init__(self, app=None):
-        dbe= config.Config.db
-        self.conn=psycopg2.connect(dbname=dbe["DATABASE_NAME"], user=dbe["DATABASE_USER"],password=dbe["DATABASE_PASS"],host=dbe["DATABASE_HOST"])        
-        self.cur = self.conn.cursor()
-
-    def commit(self):
-        self.conn.commit()
-
-    def cursor(self):
-        return self.cur
-    
-
-database = Database()
+def dbconn():
+    dbe= config.Config.db
+    conn=psycopg2.connect(dbname=dbe["DATABASE_NAME"], user=dbe["DATABASE_USER"],password=dbe["DATABASE_PASS"],host=dbe["DATABASE_HOST"])
+    return conn    
 
 class User():
 
-    def __init__(self, fname, lname,email,password):
-        super().__init__()
-        self.fname = fname
-        self.lname = lname
-        self.email = email
-        self.password = password
+    def __init__(self, fname=None, lname=None,email=None,password=None):
+        # super().__init__()
+        if fname and lname and email and password:
+            self.fname = fname
+            self.lname = lname
+            self.email = email
+            self.password = generate_password_hash(password)
     
-    #def create_user(self):
-        cursor = database.cursor()
-        cursor.execute("INSERT INTO users (fname, lname, email, password) VALUES (%s, %s, %s, %s)",
+    def create_user(self):
+        conn = dbconn()
+        cur = conn.cursor()
+        cur.execute("INSERT INTO users (fname, lname, email, password) VALUES (%s, %s, %s, %s) RETURNING id",
                                     (self.fname, self.lname, self.email, self.password))
-        database.commit()
+        id = cur.fetchone()[0]
+        conn.commit()
+        conn.close()
+        return id
+    
+    def login(self, email, password):
+        conn = dbconn()
```

Comment:
avoid creating a `database` connection for every single request since this is an expensive operation. Consider having a single point where you create a connection and use this connection throughout the life cycle of the application.